src/go/rpk/pkg/cli/cmd/acl.go

```diff
@@ -40,25 +45,41 @@ func NewACLCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 		&user,
 		&password,
 		&mechanism,
+		&enableTLS,
 		&certFile,
 		&keyFile,
 		&truststoreFile,
 		&brokers,
 	)
+	common.AddAdminAPITLSFlags(
+		command,
+		&adminAPIEnableTLS,
+		&adminAPICertFile,
+		&adminAPIKeyFile,
+		&adminAPITruststoreFile,
+	)
 
 	configClosure := common.FindConfigFile(mgr, &configFile)
 	brokersClosure := common.DeduceBrokers(
 		common.CreateDockerClient,
 		configClosure,
 		&brokers,
 	)
-	tlsClosure := common.BuildTLSConfig(&certFile, &keyFile, &truststoreFile)
+	kafkaTlsClosure := common.BuildKafkaTLSConfig(fs, &enableTLS, &certFile, &keyFile, &truststoreFile, configClosure)
+	adminTlsClosure := common.BuildAdminApiTLSConfig(
+		fs,
+		&enableTLS,
```

Comment:
I think this needs to be `adminAPIEnableTLS`

Reply:
It does! Thanks.